grammar_test/expressions.txt

```diff
@@ -72,7 +73,10 @@ i(j, 5,)
     (identifier)
     (keyword_argument (identifier) (identifier))
     (identifier)))
-  (expression_statement (call (identifier) (identifier) (number))))
+  (expression_statement (call (identifier) (identifier) (number)))
+  (expression_statement (call (call
+    (identifier) (number) (number) (number))
+		(conditional_expression (identifier) (identifier)))))
```

Comment:
@maxbrunsfeld thanks for the suggestion! I was trying to model `conditional_expression` as part of a `call` expression, but was confused why I was seeing double `call` in the AST. I've pulled `conditional_expression` out of `call` entirely in https://github.com/tree-sitter/tree-sitter-python/pull/1/commits/1b46c6100c3fa0d8df8b1da1bfdfd76c7e95c6cd, so it is modeled in the way you describe. Now the test is structured in the way we'd expect 😄